GPU/Vulkan/DrawEngineVulkan.h

```diff
@@ -194,7 +194,7 @@ class DrawEngineVulkan : public DrawEngineCommon {
 
 	void SetLineWidth(float lineWidth);
 	void SetDepalTexture(VkImageView depal) {
-		if (boundDepal_ != depal) {
+		if (boundDepal_ != depal && depal) {
```

Comment:
Hm.  Won't this prevent clearing `boundDepal_` in that case?

https://github.com/hrydgard/ppsspp/blob/9736bc431aba6525510a52a0629f95e0cf00dca4/GPU/Vulkan/DrawEngineVulkan.cpp#L464

-[Unknown]

Reply:
You're right. Skipping this change.